src/ControlSystem/ControlErrors/Translation.hpp

```diff
@@ -109,30 +111,62 @@ struct Translation : tt::ConformsTo<protocols::ControlError> {
                                          grid_position_of_A_tnsr[2]}};
     const DataVector& current_position_of_A = get<center_A>(measurements);
 
-    const DataVector rotation_error =
-        rotation_control_error_(tuner, cache, time, "Rotation", measurements);
-    // Use A because it's on the positive x-axis, however, B would work as well.
-    // Just so long as we are consistent.
-    const DataVector rotation_error_cross_grid_pos_A =
-        cross(rotation_error, grid_position_of_A);
-
-    const double expansion_error = expansion_control_error_(
-        tuner, cache, time, "Expansion", measurements)[0];
-
-    // From eq. 42 in 1304.3067
-    const quat middle_expression = datavector_to_quaternion(
-        current_position_of_A -
-        (1.0 + expansion_error / expansion_factor) * grid_position_of_A -
-        rotation_error_cross_grid_pos_A);
+    const tnsr::I<double, 3, Frame::Grid>& grid_position_of_B_tnsr =
+        Parallel::get<domain::Tags::ObjectCenter<domain::ObjectLabel::B>>(
+            cache);
+    const DataVector grid_position_of_B{{grid_position_of_B_tnsr[0],
+                                         grid_position_of_B_tnsr[1],
+                                         grid_position_of_B_tnsr[2]}};
+    const DataVector& current_position_of_B = get<center_B>(measurements);
+
+    const DataVector& grid_position_average =
+        0.5 * (grid_position_of_A + grid_position_of_B);
+    const DataVector& current_position_average =
```

Comment:
These shouldn't be references since you are computing something new. Same for the separations.